content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md

```diff
@@ -238,7 +155,7 @@ If the request specified `"owner_funds": true` and expanded transactions, the re
 * Any of the [universal error types][].
 * `invalidParams` - One or more fields are specified incorrectly, or one or more required fields are missing.
 * `lgrNotFound` - The ledger specified by the `ledger_hash` or `ledger_index` does not exist, or it does exist but the server does not have it.
-* `noPermission` - If you specified `full` or `accounts` as true, but are not connected to the server as an admin (usually, admin requires connecting on a local port).
+* `noPermission` - If you specified `full` or `accounts` as true, but are not connected to the server as an admin (usually, admin requires connecting on localhost).
```

Comment:
Isn't `full` and `accounts` deprecated? Is it possible to get `noPermission` error even without these fields?

Reply:
That's true, but deprecated does not mean removed, so it's still a real error that can occur.